bin/cli.js

```diff
@@ -131,8 +79,8 @@ yargs
   })
   .fail((err) => {
     // Prints to `stderr` by default
-    return showErrorAndQuit(err);
+    return helpers.showErrorAndQuit(err);
   })
   .argv;
 
-process.on('uncaughtException', showErrorAndQuit);
+process.on('uncaughtException', helpers.showErrorAndQuit);
```

Comment:
Good point :) Reckon it could be useful, I've updated the commands so they return a promise, allowing for a programmatic api eg.

```
const commands = require('./lib/cli/commands');

commands.dev({
  buildDir: '/tmp/landr/build',
  port: 3000
}).then((server) => {
  console.log(server);
  console.log('you just got served served');
});
```